config.yaml

```diff
@@ -19,6 +19,7 @@ laboratory: access-esm
 jobname: pre-industrial
 queue: normal
 walltime: 2:30:00
+jobfs: 1500MB
```

Comment:
Just a question, did you confirm this is correctly setting `jobfs` in the PBS submit command when doing `payu run`?

Reply:
Yep! I'm getting the following in the output logs:
``` 
JobFS requested:    1.46GB 
```